app/controllers/passwords_controller.rb

```diff
@@ -13,33 +13,34 @@ def update
         data: @recoverable.token_validation_response
       }
     else
-      render json: {"message": "Invalid token","redirect_url": "/"}, status: 422
+      render json: { 'message': 'Invalid token', 'redirect_url': '/' }, status: 422
     end
   end
 
   def create
     @user = User.find_by(email: params[:email])
     if @user
       @user.send_reset_password_instructions
-      build_response(I18n.t('messages.reset_password_success'),200)
+      build_response(I18n.t('messages.reset_password_success'), 200)
     else
-      build_response(I18n.t('messages.reset_password_failure'),404)
+      build_response(I18n.t('messages.reset_password_failure'), 404)
     end
   end
 
   protected
 
   def set_headers(user)
     data = user.create_new_auth_token
-    response.headers[DeviseTokenAuth.headers_names[:"access-token"]] = data["access-token"]
-    response.headers[DeviseTokenAuth.headers_names[:"token-type"]]   = "Bearer"
-    response.headers[DeviseTokenAuth.headers_names[:"client"]]       = data["client"]
-    response.headers[DeviseTokenAuth.headers_names[:"expiry"]]       = data["expiry"]
-    response.headers[DeviseTokenAuth.headers_names[:"uid"]]          = data["uid"]
+    headers_names = response.headers[DeviseTokenAuth.headers_names]
+    headers_names['access-token'] = data['access-token']
```

Comment:
@AlexisChavez97 This can be translated to `response.headers[DeviseTokenAuth.headers_names]['access-token']` which is wrong. headers bases on DeviseTokenAuth.headers_name.

As a result of this, the password flow is broken.